ui/home.php

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Team-229</title>
+        <style><?php foreach(glob('css/*.css') as $file) include($file); ?></style>
+        <script><?php foreach(glob('js/*.js') as $file) include($file); ?></script>
+    </head>
+    <body>
+        <?php
+        if(isset($_GET['survey'])){
```

Comment:
@peter-geek Please remove the PHP. This is meant to be a pure HTML/CSS/Javascript page